src/components/movie/delete/DeleteMovie.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classes from "./DeleteMovie.module.css";
+import DeleteText from "./text/DeleteText";
+import CloseButton from "../button/close/CloseButton";
+import ConfirmButton from "../button/confirm/ConfirmButton";
+
+function DeleteMovie(props) {
+  return (
+    <div className={classes.popup}>
+      <CloseButton action={props.toggleDeleteMovieWindow}/>
+      <DeleteText/>
+      <ConfirmButton name="CONFIRM" action={props.toggleDeleteMovieWindow}/>
```

Comment:
Данный экшен никак не меняет массив фильмов для отображения, правильно здесь получить колбэк в который прокинуть id фильма, и в верхнеуровневом управляющем компоненте мутировать данные для отображения.
```suggestion

const handleDeleteFilm = (id) => {
    props.deleteMovie(id);
    props.toggleDeleteMovieWindow()
}
return(
      <ConfirmButton name="CONFIRM" action={props.handleDeleteFilm}/>
```

Reply:
Done